src/PlutoStaticHTML.jl

```diff
@@ -37,21 +38,28 @@ using Pluto:
 using PrecompileSignatures: @precompile_signatures
 using SHA: sha256
 using TOML: parse as parsetoml
+using tectonic_jll: tectonic
+
+const PKGDIR = string(pkgdir(PlutoStaticHTML))::String
```

Comment:
This will bake the path in at build time, which won't be usable in a relocated system image. Using https://github.com/JuliaPackaging/RelocatableFolders.jl is how I'd typically solve needing to store those listings files, or just reading them in at build time into their own `String` consts.

Reply:
Nice tip. How do I read them at build time?

Reply:
For this case it would probably be something like:

```julia
const LISTINGS_DIR = RelocatableFolders.@path joinpath(@__DIR__, "listings")
```

and then wherever you need to use files from that directory at runtime you'd do

```julia
function # ...
   file = joinpath(LISTINGS_DIR, "julia_listings.tex")
end
```

and file will resolve to a file that exists regardless of where you've relocated the package to.

Reply:
(That was from memory, so not 100% sure that runs, but it's the general idea.)

If you need to access at build time then you'd just do that same thing as above, the `@path` will handle it either way.

Reply:
Thanks :+1: